exo-go/src/app_setup/index.go

```diff
@@ -0,0 +1,130 @@
+package appSetup
+
+import (
+	"io/ioutil"
+	"path"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/Originate/exosphere/exo-go/src/app_dependency_helpers"
+	"github.com/Originate/exosphere/exo-go/src/docker_compose"
+	"github.com/Originate/exosphere/exo-go/src/docker_setup"
+	"github.com/Originate/exosphere/exo-go/src/logger"
+	"github.com/Originate/exosphere/exo-go/src/os_helpers"
+	"github.com/Originate/exosphere/exo-go/src/service_config_helpers"
+	"github.com/Originate/exosphere/exo-go/src/types"
+	"github.com/Originate/exosphere/exo-go/src/util"
+)
+
+// AppSetup sets up the app
+type AppSetup struct {
+	AppConfig             types.AppConfig
+	Logger                *logger.Logger
+	DockerComposeConfig   types.DockerCompose
+	ServiceData           map[string]types.ServiceData
+	ServiceConfigs        map[string]types.ServiceConfig
+	DockerComposeLocation string
```

Comment:
I don't think this is good name. Its actually the folder we are storing `docker-compose.yml`. It also appears that its related to appDir. Thoughts on creating a function which computes this to avoid us from having to save it as a variable?

Also it appears `dockerComposeLocation` is used mostly to refer to the folder that contains `docker-compose.yml` and but is also used in dockerHelpers.GetDockerCompose as the path to the `docker-compose.yml`. Thoughts on renaming to `dockerComposeDirPath` and `dockerComposeFilePath`?